src/dini/templates/headers.js

```diff
@@ -133,7 +133,7 @@ export default conf => {
             ${showPeople(conf.authors)}
           `
         : ""}
-      ${conf.otherLinks ? conf.otherLinks.map(showLink) : ""}
+      ${conf.otherLinks ? conf.otherLinks.map(link => showLink(link)) : ""}
```

Comment:
I changed these because `showLink()`'s function signature doesn't match what `.map()` expects.

Reply:
I think this is probably not needed here, as we have control over the signature of `showLink` (it expects only one param). Issues arise when there's more than one parameter.

Reply:
yeah, true... I'll switch 'em back.